test cases/unit/106 subproject symlink/symlinked_subproject/src.c

```diff
@@ -0,0 +1,4 @@
+int foo()
```

Comment:
I think one of the CI passes is giong to get grump that this isn't `int foo(void)`